docs/metrics.md

```diff
@@ -0,0 +1,214 @@
+# Metrics {#ovms_docs_metrics}
+
+## Introduction
+
+This document describes how to use metrics endpoint in the OpenVINO Model Server. They can be applied for:
+
+- Providing performance and utilization statistics for monitoring and benchmarking purposes
+
+- Auto scaling of the model server instances in Kubernetes and OpenShift based on application related metrics
+
+> **NOTE**: Currently, metrics feature is released as a preview feature.
+
+Built-in metrics allow tracking the performance without any extra logic on the client side or using network traffic monitoring tools like load balancers or reverse-proxies.
+
+It also exposes metrics which are not related to the netowrk traffic. 
+
+For example, statistics of the inference execution queue, model runtime parameters etc. They can also track the usage based on model version, API type or requested endpoint methods.
```

Comment:
Hmm, I feel like we should have a diagram explaining these phases to the user. 
"number of active inference requests" or "execution queue size" might be unclear for the users that are not familiar with OVMS implementation